src/rpc/core_rpc_server.cpp

```diff
@@ -1147,6 +1171,240 @@ namespace cryptonote
+          {
+            case impl::type::kCommand::Configure:
+              {
+                // TODO(anonimal): Sekreta only allows to configure once,
+                //   regardless if stop'd
+                lib->configure(req.system_args);
+
+                // Create server paths
+                try
+                  {
+                    std::shared_ptr<type::kovri::PathData> rpc_data =
+                        trait->path()
+                            ->uid({uid_rpc})
+                            .type({type::kPathDirection::Server,
+                                   type::kPathAffect::HTTP})
+                            .local_point({"127.0.0.1",
+                                          {cryptonote::get_config(nettype())
+                                               .RPC_DEFAULT_PORT,
+                                           {}}})
+                            .crypto({{/* no-op */},
+                                     type::crypto::kScheme::Ed25519 /* no-op */,
+                                     "monero" + uid_rpc + ".dat"})
+                            .build();
+
+                    rpc = lib->path(rpc_data);
+                    rpc->configure();
+
+                    MGINFO_GREEN("RPC server path configured");
+
+                    // TODO(anonimal): BUG: only first path (RPC) is created at
+                    //   system level (`grep PrivateKeys` in Kovri logs)
+                    std::shared_ptr<type::kovri::PathData> p2p_data =
+                        trait->path()
+                            ->uid({uid_p2p})
+                            .type({type::kPathDirection::Server,
+                                   type::kPathAffect::Default})
+                            .local_point({"127.0.0.1",
+                                          {cryptonote::get_config(nettype())
+                                               .P2P_DEFAULT_PORT,
+                                           {}}})
+                            .crypto({{/* no-op */},
+                                     type::crypto::kScheme::Ed25519 /* no-op */,
+                                     "monero" + uid_p2p + ".dat"})
+                            .build();
+
+                    p2p = lib->path(p2p_data);
+                    p2p->configure();
+
+                    MGINFO_GREEN("P2P server path configured");
+                  }
+                catch (const ::sekreta::Exception& ex)
+                  {
+                    // TODO(unassigned): we'll want to expand command options
+                    //   for tunnel flexibility because we shouldn't need to
+                    //   stop the router upon fatal tunnel error
+                    lib->stop();
+                    sekreta->erase<api::Library>({uid_lib});
+                    LOG_ERROR(ex.what());
+                    throw;
+                  }
+
+                MGINFO_GREEN(req.system.c_str() << " configured");
```

Comment:
The "G" variants are for "use the predefined global category". You want MINFO_GREEN so it uses the rpc category from this file.